test/implementation-tests.js

```diff
@@ -12,14 +12,21 @@
 
 var jsonata = require("../src/jsonata");
 var chai = require("chai");
+var chaiAsPromised = require("chai-as-promised");
+chai.use(chaiAsPromised);
 var expect = chai.expect;
 
 var testdata1 = {
-    "foo": {
-        "bar": 42,
-        "blah": [{"baz": {"fud": "hello"}}, {"baz": {"fud": "world"}}, {"bazz": "gotcha"}],
-        "blah.baz": "here"
-    }, "bar": 98
+    foo: {
+        bar: 42,
+        blah: [
+            { baz: { fud: "hello" } },
+            { baz: { fud: "world" } },
+            { bazz: "gotcha" },
+        ],
+        "blah.baz": "here",
+    },
+    bar: 98,
```

Comment:
I feel like I focus on the wrong thing, but I feel like the prettier changes in this file can make our life harder later on when we try to merge this back